DRX/Fringing/fringeBeam.py

```diff
@@ -279,6 +282,8 @@ def main(args):
                         help='dipole number')
     parser.add_argument('filename', type=str, nargs='+',
                         help='filename to process')
+    parser.add_argument('-S','--lwasv', action='store_true',
+                        help='Station is LWA-SV')
```

Comment:
I usually use either `-s` (lower cased) if I can or a `-v` to switch over to LWA-SV in a script.